hvplot/converter.py

```diff
@@ -215,7 +216,8 @@ def __init__(self, data, x, y, kind=None, by=None, use_index=True,
                  crs=None, fields={}, groupby=None, dynamic=True,
                  width=700, height=300, shared_axes=True,
                  grid=False, legend=True, rot=None, title=None,
-                 xlim=None, ylim=None, clim=None, xticks=None, yticks=None,
+                 xlim=None, ylim=None, clim=None, center=None,
```

Comment:
I'd say `center` should be called `symmetric` which is the equivalent keyword in holoviews.